### ISSUE TITLE
Tapping on disconnected watch does nothing

### ISSUE THREAD

**Issue by matejdro** [pebble-dev/mobile-app#101]
This has been since discovered as not a bug, but let's use this ticket as discussion point. Continuing discussion from #97

**Comment by matejdro** [comment]
> That seems like unexpected behavior. I don't think users who do understand the design will expect that. I think a better option would be to tell the user with a popup or toast notification "can't disconnect from watch since we aren't connected"or something along those lines.

Why would user think that tapping on that watch would disconnect it? There is a clear visual distinction between connected watch and all other watches.

It seems reasonable to me that tapping on connected watch has different action than tapping on disconnected one.

**Comment by AliceGrey** [comment]
> > That seems like unexpected behavior. I don't think users who do understand the design will expect that. I think a better option would be to tell the user with a popup or toast notification "can't disconnect from watch since we aren't connected"or something along those lines.
> 
> Why would user think that tapping on that watch would disconnect it? There is a clear visual distinction between connected watch and all other watches.
> 
> It seems reasonable to me that tapping on connected watch has different action than tapping on disconnected one.

I think I may have misunderstood the original issue.

**Comment by matejdro** [comment]
Right, original issue is meant for watches below the line. So when you would tap on the name of the disconnected watches, it would start connecting.

It would be a good idea to add a warning before disconnecting though, if there is existing watch already connecting. Like "Connecting to X will disconnect from Y. Proceed?".
